docs/docs/development/setup/native.mdx

```diff
@@ -0,0 +1,318 @@
+
+Next, you'll need to clone the ZMK source repository if you haven't already. Open a terminal and navigate to the folder you would like to place your `zmk` directory in, then run the following command:
+
+```sh
+git clone https://github.com/zmkfirmware/zmk.git
+```
+
+Then step into the repository.
+
+```sh
+cd zmk
+```
+
+## 3. Get Zephyr and install Python dependencies
+
+:::note
+These steps are very similar to Zephyr's [Get Zephyr and install Python dependencies](https://docs.zephyrproject.org/3.5.0/develop/getting_started/index.html#get-zephyr-and-install-python-dependencies) instructions, but specialized for ZMK.
+:::
+
+<EnvTabs>
+<TabItem value="venv">
+<Tabs groupId="operating-systems" defaultValue="ubuntu">
+  <TabItem value="ubuntu" label="Ubuntu">
+
+1. Use `apt` to install Python `venv` package:
+
+```sh
+sudo apt install python3-venv
+```
+
+2. Create a new virtual environment and activate it:
+
+```sh
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
+</TabItem>
+  <TabItem value="win" label="Windows">
+1. Create a new virtual environment:
+
+```sh
+python -m venv .venv
+```
+
+2. Activate the virtual environment:
+
+<WinTermTabs>
+<TabItem value="cmd">
+
+```sh
+.venv\Scripts\activate.bat
+```
+
+</TabItem>
+<TabItem value="ps">
+
+```sh
+.venv\Scripts\Activate.ps1
+```
+
+</TabItem>
+</WinTermTabs>
+</TabItem>
+ <TabItem value="mac" label="Mac OS">
+
+1. Create a new virtual environment:
+
+```sh
+python3 -m venv .venv
+```
+
+2. Activate the virtual environment:
+
+```sh
+source .venv/bin/activate
+```
+
+</TabItem>
+</Tabs>
+
+Once activated your shell will be prefixed with `(.venv)`. The virtual environment can be deactivated at any time by running `deactivate`.
+
+:::note
+Remember to activate the virtual environment every time you start working.
+:::
+
+4. Install west:
+
+```sh
+pip install west
+```
+
+5. Initialize the application and update to fetch modules, including Zephyr:
+
+```sh
+west init -l app/
+west update
+```
+
+:::tip
+This step pulls down quite a bit of tooling, be patient!
+:::
+
+6. Export a [Zephyr CMake package](https://docs.zephyrproject.org/3.5.0/build/zephyr_cmake_package.html#cmake-pkg). This allows CMake to automatically load boilerplate code required for building Zephyr applications.
+
+```sh
+west zephyr-export
+```
+
+7. Install the additional dependencies found in Zephyr's `requirements-base.txt`:
+
+```sh
+pip install -r zephyr/scripts/requirements-base.txt
+```
+
+</TabItem>
+<TabItem value="glob">
+<Tabs groupId="operating-systems" defaultValue="ubuntu">
+  <TabItem value="ubuntu" label="Ubuntu">
+1. Install `west`, and make sure `~/.local/bin` is on your `PATH` environment variable:
+
+```sh
+pip3 install --user -U west
+echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
+source ~/.bashrc
+```
+
+
+</TabItem>
+  <TabItem value="win" label="Windows">
+1. Install `west`, and make sure `~/.local/bin` is on your `PATH` environment variable:
+
+```sh
+pip install -U west
+```
```

Comment:
````suggestion
1. Install `west`:

```sh
pip install -U west
```

Make sure the Python script directory is on your `PATH` environment variable by opening a PowerShell window and running the following commands:

```powershell
$Scripts = python -c "import sysconfig; print(sysconfig.get_path('scripts'))"
$Path = [Environment]::GetEnvironmentVariable('PATH', 'User')
[Environment]::SetEnvironmentVariable('PATH', "$Path;$Scripts", 'User')
$env:PATH += ";$Scripts"
```

````

I realized we lost the info in https://zmk.dev/docs/development/setup#install-west Windows tab on the PATH stuff here, even though it is mentioned in the bullet point.

(This should be under bullet 1., I am not sure the way I wrote it above works)